.github/workflows/go-audit.yml

```
@@ -0,0 +1,66 @@
+
+    # Official Go vulnerability scanner
+    - name: Go vulnerability database check
+      working-directory: ./cla-backend-legacy
+      run: |
+        go install golang.org/x/vuln/cmd/govulncheck@latest
+        govulncheck -json ./... > govulncheck-results.json
+        govulncheck ./...
+      continue-on-error: true
+
+    - name: Upload vulnerability results
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.11)

[error] the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)